spec.html

```diff
@@ -35021,10 +35124,13 @@ <h1>%TypedArray%.from ( _source_ [ , _mapfn_ [ , _thisArg_ ] ] )</h1>
         </emu-alg>
 
         <emu-clause id="sec-iterabletolist" aoid="IterableToList">
-          <h1>Runtime Semantics: IterableToList ( _items_, _method_ )</h1>
-          <p>The abstract operation IterableToList takes arguments _items_ and _method_. It performs the following steps when called:</p>
+          <h1>Runtime Semantics: IterableToList ( _items_ [ , _method_ ] )</h1>
+          <p>The abstract operation IterableToList performs the following steps:</p>
```

Comment:
The AO frontmatter was recently made consistent across all AOs, so please keep this paragraph format the same as before, except changing referring to _method_ as an optional argument.